server/routes/address.js

```diff
@@ -57,6 +57,17 @@ module.exports = ({ licenceService, nomisPushService }) => (router, audited, { p
       const { enterAlternative, bookingId } = req.body
       const { licence } = res.locals.licence
 
+      const errorObject =
+        licenceService.validateForm({
+          formResponse: req.body,
+          pageConfig: formConfig.rejected,
+        }) || {}
+
+      if (errorObject.enterAlternative) {
```

Comment:
Not sure if you are following a pattern but is there a reason to be specific about the type of error?  

Would it be valid to just check if there are *any* errors here?

Reply:
I did start off by just checking for the errorObject but always got an entry for csrf and bookingId included in the errorObject, like this:
{ 
   enterAlternative: 'Select yes or no',
  _csrf: '"_csrf" is not allowed',
  bookingId: '"bookingId" is not allowed' 
}

So, I tried removing crsf and bookingId from server/views/includes/alternativeForm.pug but that caused the app to route to http://localhost:8080/auth/login

Hence I thought I should only do a conditional check for errorObject.enterAlternative

So, going back to your suggestion, I will not specifically check for the enterAlternative in the error Object but rather filter out the csrf and bookingId.